addons/auth_signup/static/src/js/signup.js

```diff
@@ -17,6 +17,9 @@ publicWidget.registry.SignUpForm = publicWidget.Widget.extend({
      */
     _onSubmit: function () {
         var $btn = this.$('.oe_login_buttons > button[type="submit"]');
+        if ($btn.prop("disabled")) {
+            return;
+        }
```

Comment:
I believe since latest changes this became useless. At least I removed it locally and tried to submit using the enter keystore and it didn't sent.